cross-media-measurement/src/test/cc/wfa/measurement/common/crypto/liquid_legions_v2_encryption_utility_test.cc

```diff
@@ -574,21 +577,19 @@ TEST(CompleteSetupPhase, NoiseSumAndMeanShouldBeCorrect) {
   public_key.set_generator(public_key_pair.first);
   public_key.set_element(public_key_pair.second);
 
-  int publisher_count = 3;
-
   int64_t computed_blinded_histogram_noise_offset = 5;
   int64_t computed_publisher_noise_offset = 4;
   int64_t computed_reach_dp_noise_offset = 3;
   int64_t expected_total_register_count =
       computed_publisher_noise_offset * 2 + computed_reach_dp_noise_offset * 2 +
-      computed_blinded_histogram_noise_offset * publisher_count *
-          (publisher_count + 1);
+      computed_blinded_histogram_noise_offset * kNumOfPublishers *
```

Comment:
While you're here, maybe rename kNumOfPublishers to kPublisherCount? That might be a little more clear?

Reply:
Done. Also renamed kNumOfWorkers to kWorkerCount.